apps/condo/lang/ru/ru.json

```diff
@@ -2392,6 +2392,9 @@
   "profile.ChangePasswordModalTitle": "Изменение пароля",
   "profile.ChangePassword": "Сбросить пароль",
   "profile.PasswordResetTimeout": "(сбросить ещё раз можно через 00: {countdown} секунд)",
+  "profile.Delete.CurrentUser": "Удалить профиль",
+  "profile.Delete.ConfirmMessage": "Вы уверены, что хотите удалить профиль? Восстановить профиль будет невозможно",
```

Comment:
Is it worth mentioning here that the organization's data will not be deleted?

Reply:
Hmmm, I think, that name of button "Delete Profile", says users about delete only his profile user. However, I can easily fix this.

Reply:
You can do it as you see fit :)